ex3/ex3.js

```diff
@@ -7,3 +7,62 @@
  * containsValue([1,3,5,7,9], 7) === true;
  * containsValue([9,7,5,3,1], 7) === true;
  */
+function containsValue(arrayNumber, searchForNumber) {
+
+	console.assert(isNumber(searchForNumber),'is not a number');
+	//  We define isNumberInArray as false so only if we find the search number we will set it to true 
+	//  otherwise will return falsh
+	var isNumberInArray = false;
+	
+	//  Loop through arrayNumber 
```

Comment:
This comment is trivial (i.e. it doesn't provide NEW information than just reading the line)